cobalt/android/apk/app/src/main/java/dev/cobalt/coat/javabridge/AmatiDeviceInspector.java

```diff
@@ -0,0 +1,34 @@
+package dev.cobalt.coat.javabridge;
+
+import static dev.cobalt.util.Log.TAG;
+
+import android.content.Context;
+import android.util.Log;
+
+/**
+ * A simple example of implement CobaltJavaScriptAndroidObject.
```

Comment:
Either "to implement" or "of implementing".

Reply:
Also a minimal description is needed here, example:

```
 * A simple example of implement CobaltJavaScriptAndroidObject.
 * Allows querying the platform for whether the device supports
 * GoogleTV (Googles see go/what-is-amati).
```

Reply:
updated